docs/sources/community/lids/0001-Introduction.md

```diff
@@ -2,8 +2,8 @@
 title: "0001: Introducing LIDs"
 description: "0001: Introducing LIDs"
 aliases: 
-- /docs/loki/latest/lids
-- /docs/loki/latest/community/lids
+- /docs/loki/latest/lids/0001-introduction/
```

Comment:
We must not have absolute aliases in versioned documentation unless you want to have to update the version every time a new release/version branch is cut. Old release branches that still have "latest" in the aliases steal redirects if the page is moved later. This takes readers to old documentation unexpectedly when they follow indexed search engine results.

The correct alias to redirect http://localhost:3002/docs/loki/latest/lids/0001-introduction/ to this page is:

```suggestion
- ../../lids/0001-introduction/
```

Reply:
I had noted in the description that "I could not get relative redirects to work for the LIDS or Maintaining files, so had to use the older format, which did work."  I don't know why it wasn't working last week (I tested repeatedly and couldn't get the redirects to work), because it's working fine now.  ¯\_(ツ)_/¯   Updated the five files that still used absolute references to use the new format.